HotelPriceScout/Pages/DashboardPage.razor

```diff
@@ -238,94 +243,17 @@
 
 @code
 {
-    
+
     private void RemoveCard(WarningMessage popup)
     {
         Dashboard.WarningMessage.Remove(popup);
     }
-    public bool thermometerIsOpen = false;
+    private bool dropdown = false;
```

Comment:
Private
![artworks-kcvrHMUw9R4z61cb-jiXwVg-t500x500](https://user-images.githubusercontent.com/72670627/144223771-17fa74f1-647f-4d5d-b1a4-c203ddbf9e08.jpg)

Reply:
u love it